.github/workflows/schedule-daily.yml

```
@@ -0,0 +1,21 @@
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write # Good practice to check for issue write permission.
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-pr-label: "stale"
+          stale-pr-message: "This Pull Request is stale because it has been open for 14 days with no activity. If you want to keep this Pull Request open, add a comment or update the branch."
+          days-before-stale: 14
+          dry-run: true
```

actionlint (1.7.7)

input "dry-run" is not defined in action "actions/stale@v9". available inputs are "any-of-issue-labels", "any-of-labels", "any-of-pr-labels", "ascending", "close-issue-label", "close-issue-message", "close-issue-reason", "close-pr-label", "close-pr-message", "days-before-close", "days-before-issue-close", "days-before-issue-stale", "days-before-pr-close", "days-before-pr-stale", "days-before-stale", "debug-only", "delete-branch", "enable-statistics", "exempt-all-assignees", "exempt-all-issue-assignees", "exempt-all-issue-milestones", "exempt-all-milestones", "exempt-all-pr-assignees", "exempt-all-pr-milestones", "exempt-assignees", "exempt-draft-pr", "exempt-issue-assignees", "exempt-issue-labels", "exempt-issue-milestones", "exempt-milestones", "exempt-pr-assignees", "exempt-pr-labels", "exempt-pr-milestones", "ignore-issue-updates", "ignore-pr-updates", "ignore-updates", "include-only-assigned", "labels-to-add-when-unstale", "labels-to-remove-when-stale", "labels-to-remove-when-unstale", "only-issue-labels", "only-labels", "only-pr-labels", "operations-per-run", "remove-issue-stale-when-updated", "remove-pr-stale-when-updated", "remove-stale-when-updated", "repo-token", "stale-issue-label", "stale-issue-message", "stale-pr-label", "stale-pr-message", "start-date"

(action)